packages/admin/admin-generator/src/commands/generate/generateGrid/generateGrid.ts

```diff
@@ -148,15 +150,16 @@ const getValueOptionsLabelData = (messageId: string, label: string | StaticSelec
     };
 };
 
-export function generateGrid(
+type GridConfigRow = { __typename: string } & GridValidRowModel;
+export function generateGrid<Row extends GridConfigRow>(
```

Comment:
I'd suggest to use `<Config extends GridConfig>` instead, wouldn't this be more fitting?